README.md

```diff
@@ -200,53 +375,120 @@ a `Close` can be sent by Alice _or_ the PFI as a reply to an RFQ or a Quote
+| field    | data type                                   | required | description                                               |
+| -------- | ------------------------------------------- | -------- | --------------------------------------------------------- |
+| `payin`  | [`PaymentInstruction`](#paymentinstruction) | N        | Link or Instruction describing how to pay the PFI.        |
+| `payout` | [`PaymentInstruction`](#paymentinstruction) | N        | Link or Instruction describing how to get paid by the PFI |
+
+##### `PaymentInstruction`
+| field         | data type | required | description                                                               |
 | ------------- | --------- | -------- | ------------------------------------------------------------------------- |
 | `link`        | String    | N        | Link to allow Alice to pay PFI, or be paid by the PFI                     |
 | `instruction` | String    | N        | Instruction on how Alice can pay PFI, or how Alice can be paid by the PFI |
 
 
 ```json
 {
-  "expiryTime": "2023-04-14T12:12:12Z",
-  "totalFee": 200,
-  "amount": 383,
-  "paymentInstructions": {
-    "payin": {
-      "link": "square-pay.com?for=alice&amount=10"
-    }
-  }
+  "metadata": {
+    "from": "did:ex:pfi",
+    "to": "did:ex:alice",
+    "kind": "quote",
+    "id": "abcd123",
+    "threadId": <RFQ_ID>,
+    "parentId": <RFQ_ID>,
+    "createdAt": "ISO_8601"
+  },
+  "data": {
+    "expiresAt": "ISO_8601",
+    "paymentInstructions": null,
+    "base": {
+      "currencyCode": "BTC",
+      "amountSubunits": "3",
+      "feeSubunits": "0",
+    },
+    "quote": {
+      "currencyCode": "USD",
+      "amountSubunits": "100_00",
+      "feeSubunits": "0",
+    },
+  },
+  "signature": "COMPACT_JWS",
 }
 ```
 
 
-## `OrderStatus`
+#### `Order`
+> Alice -> PFI: I'm happy with the quote and I want to execute the transaction"
+
+
+```json
+{
+  "metadata": {
+    "from": "did:ex:alice",
+    "to": "did:ex:pfi",
+    "kind": "order",
+    "id": "abcd123",
+    "threadId": <RFQ_ID>,
+    "parentId": <QUOTE_ID>,
+    "createdAt": "ISO_8601"
+  },
+  "data": {},
+  "signature": "COMPACT_JWS",
+}
+```
+
+#### `OrderStatus`
 > PFI -> Alice: "Here's the status of your order."
 
+
+> **Note**
+> There can be many `OrderStatus` messages in a tbdex thread. 1 for every status change that a PFI wants to inform Alice of.
+
 | field         | data type | required | description                                   |
 | ------------- | --------- | -------- | --------------------------------------------- |
-| `orderStatus` | enum      | Y        | Current status of Order that's being executed |
+| `orderStatus` | string    | Y        | Current status of Order that's being executed |
+
 
 ```json
 {
-  "orderStatus": "PENDING"
+  "metadata": {
+    "from": "did:ex:pfi",
+    "to": "did:ex:alice",
+    "kind": "orderStatus",
+    "id": "abcd123",
+    "threadId": <RFQ_ID>,
+    "parentId": <ORDER_ID> || <ORDER_STATUS_ID>,
```

Comment:
i thought there could be N orderstatus against 1 order, so the parent was presumed to be OrderId, not OrderStatusId?

Reply:
Yeah the exchange can be like
```
Alice --order-->   PFI
PFI --status 1--> Alice
PFI --status 2--> Alice
```

For status 1, parentId will be the order ID. For status 2, parentId will be status 1's id.